app/models/repo.js

```diff
@@ -42,6 +43,16 @@ const Repo = Model.extend({
   }),
   _branches: hasMany('branch'),
 
+  // We do not include an error state as part of an API response. This is only
+  // transmitted via a Pusher message for now.
```

Comment:
I don't understand this comment. Even if an error state is not included in an API response, wouldn't it be updated when a pusher message comes in? Ie. I'm not sure what are the implications of that comment.

Reply:
Deleted in 59a31110.